hnn_core/extracellular.py

```diff
@@ -557,17 +558,25 @@ def _get_nrn_times(self):
         else:
             raise RuntimeError('Simulation not yet run!')
 
-    def _calc_potentials_callback(self):
-        # keep all data in Neuron objects for efficiency
 
-        # 'gather' the values of seg.i_membrane_ into self.imem_vec
-        self._nrn_imem_ptrvec.gather(self._nrn_imem_vec)
+def _calc_potentials_callback(nrn_arr):
```

Comment:
I know `extra_scatter_gather` expects a callback but for a Python literate, it's probably clear that this is a callback. How about naming this so that it tells what it does? E.g.,

`_gather_nrn_voltages`

?